### PR TITLE
[LimitOrder] Refactor test tokens

### PR DESCRIPTION
**Pull request by anxolin**
Small refactor on some repetitive mock token declarations we were using in many stories

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr1386--dexreact.review.gnosisdev.com

File: src/components/trade/LimitOrder/LimitOrder.stories.tsx [hunk 1]
```diff
@@ -1,41 +1,31 @@
...
-  receiveToken: DAI,
+  sellToken: DAI,
+  receiveToken: GNO,
   limitPrice: new BigNumber('55.247234'),
   amount: '100',
-  isPriceInverted: false,
 }
 
 const Template: Story<Partial<Props>> = (props) => {
+  const [isPriceInverted, setIsPriceInverted] = useState<boolean>(false)
```
Comment (W3stside) [inline src/components/trade/LimitOrder/LimitOrder.stories.tsx:21]: is `boolean` type really required here?

File: src/components/trade/LimitOrder/LimitOrder.stories.tsx [hunk 2]
```diff
@@ -1,41 +1,31 @@
...
+
   return (
     <LimitOrder
+      isPriceInverted={isPriceInverted}
       onSwapPrices={(): void => {
         console.log('[LimitOrder.story] Swap Prices')
+        setIsPriceInverted(!isPriceInverted)
```
Comment (W3stside) [inline src/components/trade/LimitOrder/LimitOrder.stories.tsx:28]: `setIsPriceInverted(state => !state)`

**Review by W3stside — APPROVED** [review]

File: src/components/trade/LimitOrder/LimitOrder.stories.tsx (hunk 1 above)
Comment (anxolin) [reply src/components/trade/LimitOrder/LimitOrder.stories.tsx:21]: nup, good point, can be implied

File: src/components/trade/LimitOrder/LimitOrder.stories.tsx (hunk 2 above)
Comment (anxolin) [reply src/components/trade/LimitOrder/LimitOrder.stories.tsx:28]: same thing, isn't it?
I'll use yours, since it's more "pure"

**Review by Velenir — APPROVED** [review]

**Pull request merged**